### PR TITLE
Anonymous Message - PR for issue #119

### PR DESCRIPTION
**Pull request by BeaulieuAdrien**
Anonymous Message - PR for issue #119

### CODE REVIEW

**Review by NariieL — APPROVED** [review]
Admin Approve.

File: resources/[qb]/qb-core/server/functions.lua [hunk 1]
```diff
@@ -46,7 +46,7 @@ function QBCore.Functions.GetUserAccount(source)
         local p = promise.new()
         local resolved = false
 
-        MySQL.single("SELECT a.* FROM soz_api.accounts a LEFT JOIN soz_api.account_identities ai ON a.id = ai.accountId WHERE a.whitelistStatus = 'ACCEPTED' AND ai.identityType = 'STEAM' AND ai.identityId = ? LIMIT 1", {steam}, function(result)
+        MySQL.single("SELECT a.* FROM soz_database.account a WHERE a.licence = ? LIMIT 1", {steam}, function(result)
```
Comment (joelwurtz) [inline resources/[qb]/qb-core/server/functions.lua]: why ? this should not be modified here, please revert this change to your PR

File: resources/[soz]/soz-utils/client/discord.lua [hunk 2]
```diff
@@ -1,39 +1,7 @@
...
-        --[[ 
-            Here you can add buttons that will display in your Discord Status,
-            First paramater is the button index (0 or 1), second is the title and 
-            last is the url (this has to start with "fivem://connect/" or "https://") 
-        ]] --
-
-        SetDiscordRichPresenceAction(0, "Rejoindre SOZ", "https://discord.gg/soz-pas-soz")
-
-        -- It updates every minute just in case.
+
```
Comment (joelwurtz) [inline resources/[soz]/soz-utils/client/discord.lua]: Same this should not be modified, only add files relevant to this PR please

**Comment by joelwurtz** [comment]
Thanks for this PR, can you remove the unwanted files modification in this PR and squash your commit if possible ? We would gladly accept this PR with that done, ty

**Comment by BeaulieuAdrien** [comment]
> Thanks for this PR, can you remove the unwanted files modification in this PR and squash your commit if possible ? We would gladly accept this PR with that done, ty

Sorry, I don't know how this happened. It is now fixed.

**Comment by joelwurtz** [comment]
Ca a été mergé ref https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/commit/34094e084d8f2a393bab5690eaf25d93e7a3d5fb merci énormement ❤️

**Pull request closed without merging**